### PR TITLE
Tracks/shimmer

### LINKED ISSUE #649: Add Shimmer Layout for Learning Tracks
**Describe the feature you'd like**

**Screenshots**

**Additional context**

**Would you like to work on the issue?**

- [ ] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by iamspathan**
Fixes #649 

Changes: added shimmer in In All Learning Tracks and Learning Tracks Course

Screenshot: 
gif

gif

### CODE REVIEW

File: app/src/main/java/com/codingblocks/cbonlineapp/tracks/LearningTracksActivity.kt [hunk 1]
```diff
@@ -63,15 +57,19 @@ class LearningTracksActivity : BaseCBActivity() {
         setToolbar(tracksToolbar)
         setUpBottomSheet()
         setChips()
+        activity_learning_shimmer.showShimmer(true)
```
Comment (aggarwalpulkit596) [inline app/src/main/java/com/codingblocks/cbonlineapp/tracks/LearningTracksActivity.kt]: use camelcase

File: app/src/main/res/layout/item_skeleton_track_course_card.xml [hunk 2]
```diff
@@ -0,0 +1,123 @@
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (aggarwalpulkit596) [inline app/src/main/res/layout/item_skeleton_track_course_card.xml]: use material card view

**Review by aggarwalpulkit596 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/item_skeleton_track_course_card.xml (hunk 2 above)
Comment (iamspathan) [reply app/src/main/res/layout/item_skeleton_track_course_card.xml]: Changes made in the last commit

File: app/src/main/java/com/codingblocks/cbonlineapp/tracks/LearningTracksActivity.kt (hunk 1 above)
Comment (iamspathan) [reply app/src/main/java/com/codingblocks/cbonlineapp/tracks/LearningTracksActivity.kt]: Changes made in the last commit

**Comment by aggarwalpulkit596** [comment]
@iamspathan always add a screenshot for UI related issues

**Comment by iamspathan** [comment]
> @iamspathan always add a screenshot for UI related issues
Sure.  Screenshot added

**Pull request merged**